server/src/model.ts

```diff
@@ -2455,3 +2457,55 @@ export async function getFolderContent({
     notMe,
   };
 }
+
+// TODO: Currently only accesses your own content. Do we want to change this?
+export async function getParentFolder(id: number, loggedInUser: number) {
```

Comment:
Do we want this as an API endpoint? I noticed that model.ts has getActivity() which includes name, and parent id, perhaps we could turn that into an API endpoint?

Reply:
`getActivity()` only works for activities, not folders, so it wouldn't work here. Also, it should not be turned into an api as it includes `sortIndex`, which isn't serializable. We could make this something like `getContentInfo()` and return a few more fields, if it is possible that it will be reused in other contexts.